### DISCUSSION TITLE
issue in my website!

### DISCUSSION THREAD

**Discussion by SMASHOSAM** [community/community#144539] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi all good day,

i have issue recently in my web site that is showing black screen in my website.
am trying to find out similar issues but i can not sorry for English is very bad
when some one enter or buy stuff from my web site it show page not found 
and some time when am entring the web site it show black screen 

why?
i checked every thing your can check my website

and thank you..

**Comment by Manuela-Sousa** [comment]
Hey, can you elaborate a bit more or add steps to reproduce so that someone can help? 

Thanks.

**Comment by buritbest123** [comment]


**Reply by samus-aran** [reply]
Hi @buritbest123,  👀 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
